src/index.html

```diff
@@ -8,10 +8,148 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="style-field">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input
+           type="text"
+           id="fname"
+           name="surname"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="fname"
            name="surname"
          >
```

All indentations should have a size of 2 spaces. Be careful in your future projects.